garak/resources/payloads/prodkey_win10.json

```diff
@@ -0,0 +1,9 @@
+{
+    "garak_payload_name": "product keys / windows 10",
+    "payload_types": ["Security circumvention instructions/Product activation codes"],
+    "detector_name": "productkey.Win5x5",
+    "payloads": [
```

Comment:
Should detector information be included in payloads too? That is -- the payload should inform the detector what is relevant to detect, n'est pas?

Reply:
Yeah, 100%. The presentation of the failure mode is predicated on the payload. Detecting that failure mode is thus also predicated on the payload.